lib/constable_web/templates/announcement/_form.html.eex

```diff
@@ -13,13 +13,15 @@
   </nav>
   <div class="col-markdown announcement-create active">
     <div class="announcement-form">
+      <%= label f, :title, "Announcement title:" %>
```

Comment:
Minor thing, but I’d prefer to remove the colons to be consistent across our thoughtbot apps.

```suggestion
      <%= label f, :title, "Announcement title" %>
```